build.assets/api/update-module-path.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+if [ -z "$1" ]
+    then echo "Usage: $(basename $0) <version>" 1>&2; exit 1
+fi
+
+VERSION=$1
+MAJ_VER=$(echo ${VERSION} | cut -d "." -f1)
+MAJ_VER_SUF=$( if [ "${MAJ_VER}" -ge 2 ]; then echo "/v${MAJ_VER}"; else echo ""; fi )
```

Comment:
If the script is kept, I tend to prefer explicit variable names rather than shortened ones for clarity. Using the shorter versions doesn't really gain anything and reduces glance value.

```suggestion
MAJOR_VERSION=$(echo ${VERSION} | cut -d "." -f1)
MAJOR_VERSION_SUFFIX=$( if [ "${MAJOR_VERSION}" -ge 2 ]; then echo "/v${MAJOR_VERSION}"; else echo ""; fi )
```